src/shipit_code_coverage/tests/conftest.py

```diff
@@ -71,6 +75,26 @@ def GROUP_TASKS_2():
     return load_json('task-group_2.json')
 
 
+@pytest.fixture(scope='session')
+def MERCURIAL_COMMIT_RESPONSE():
+    return load_file('rev_hg_0d1e55d87931fe70ec1d007e886bcd58015ff770')
+
+
+@pytest.fixture(scope='session')
+def GITHUB_COMMIT_RESPONSE():
+    return load_file('rev_git_876c7dd30586f9c6f9c99ef7444f2d73c7acfe7c')
+
+
+@pytest.fixture(scope='session')
+def MERCURIAL_COMMIT():
```

Comment:
Define a single fixture for get_commit and a single fixture for get_mercurial and call responses.add there (no need for separate support files, putting everything in the fixture makes it clearer given the small size of the response)

Reply:
Done, let me know if it's what you expected